gsgmail_connector.py

```diff
@@ -407,6 +338,137 @@ def _handle_run_query(self, param):
 
         return action_result.set_status(phantom.APP_SUCCESS)
 
+    def _body_from_part(self, part):
+        charset = part.get_content_charset() or "utf8"
```

Comment:
Isn't it 'utf-8'?

Reply:
It is, checked with emai lib charsets. All of that are left overs from previous implementation